zubhub_frontend/zubhub/src/assets/js/styles/views/search_results/searchResultsStyles.js

```diff
@@ -91,4 +94,51 @@ const styles = theme => ({
   },
 });
 
-export default styles;
+export const resultModal = (theme) => ({
+  root: {
+    maxWidth: '1152px',
```

Comment:
use clamp, this is gonna affect responsiveness.

Try to avoid setting width and height with px.